tests/TextDiffTest.php

```diff
@@ -24,8 +24,6 @@
  * @internal
  *
  * @coversNothing
- *
- * @group covers-nothing
```

Comment:
i'm not convinced. Isn't it changing the logic?
like, for coverage job, on master we do not test this file, but on your PR we do?

Reply:
Exactly, it's an example of what I said [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7611#pullrequestreview-1794057188).

Reply:
> for coverage job, on master we do not test this file, but on your PR we do?

why would you want to run this file in a coverage job?

Reply:
I would not like to. but if I read this PR properly, on your PR this file is getting executed in CI-code-coverage job

Reply:
> on your PR this file is getting executed in CI-code-coverage job

how did you come to this conclusion? How do you read the changes in `phpunit.xml.dist` then?

Reply:
looks like I overlooked changes in that file - not the best from me, multiple files 36 files affected overloaded me, apparently ;)

but hey, that's exactly why I asked "isn't it changing logic?" or "(what) on your PR we do?" and not "this is wrong, go away".

Reply:
I've opened this PR with, I thought, quite a straightforward change and clear description, and got two answers starting with "i'm not convinced" and "I am not sure if it's a good direction". Both are not encouraging, don't you agree?

Reply:
One person asking for help to understand "how this PR works?", while other was wondering if it makes sense to forbid whole feature.
They serve different purpose, but not of them is harshly pushing "go away", but opening the doors for better understanding or alignment.